README.md

```diff
@@ -1,11 +1,73 @@
 # README
 
-Add your handler readme here. Remember to include the following:
+## What is this?
 
-- Tell people how to install it (e.g. pip install ...).
-- Be clear about the purpose of the template handler, its capabilities and limitations.
-- Tell people how to use it.
-- Give examples of the template handler in use.
+`sceptre-cdk-handler` is a TemplateHandler for Sceptre (versions 2.7 and up) that lets you use a
+Python AWS CDK template as a stack's template.
 
-If you have any questions or encounter an issue
-[please open an issue](https://github.com/Sceptre/project/issues/new)
+This template handler will use the AWS CDK to synthesize the CDK stack into a CloudFormation template
+and then run `npx cdk-assets` to publish any required assets to S3/ECR.
+
+**By using the CDK Handler, you are letting CDK synthesize a template, and upload artifacts to S3 and ECR
+and then using Sceptre to actually do the deployment of the template to a stack.**
+In other words, by using this handler with Sceptre, _you skip ever using `cdk deploy`; It's not needed_.
+
+By using this handler, you can now use CDK templates with all your favorite Sceptre commands, like
+`launch`, `validate`, `generate`, and `diff` (along with all the rest)!
+
+## How to install sceptre-cdk-handler
+
+Simply `pip install sceptre-cdk-handler`. **Additionally, you need the [AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html) installed along with its prerequisites and the [CDK Assets](https://www.npmjs.com/package/cdk-assets) NPM packacge**.
+
+## How to use sceptre-cdk-handler
+
+The template "type" for this handler is `cdk`. 
+
+### Arguments:
+
+* `path` (string, required): The path to the CDK template.
+* `context` (object, optional): The context for the CDK Stack. See [CDK Context](https://docs.aws.amazon.com/cdk/v2/guide/context.html) for further info.
+
+#### CDK Template Requirements
+
+1. The CDK template must only contain one CDK Stack class, and the class must be called `CdkStack`. 
+
+#### CDK Feature Flags and Custom Bootstrap
+
+To set any [CDK Feature Flags](https://docs.aws.amazon.com/cdk/v2/guide/featureflags.html) or to specify a modified
+[CDK Bootstrap](https://docs.aws.amazon.com/cdk/v2/guide/bootstrapping.html) qualifier, set these in the handler's
+`context` argument. See [sceptre-example](sceptre-example) for an example of this.
```

Comment:
It might be helpful to add a reminder that the `template` argument on a stack config _is_ resolvable, even recursively so, so users can use resolvers within the context object to "wire in" other resolvable values.